### PR TITLE
Add crop variants

### PR DESCRIPTION
**Pull request by h3nn3s**
This PR adds documentation about crop variants to TYPO3 Explained docu.

### CODE REVIEW

File: Documentation/ApiOverview/CropVariants/General/Index.rst [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+.. _cropvariants:
+
+=====================
+General Configuration
+=====================
+
+The following examples are basically meant to add one single
+cropping configuration to sys_file_reference,  which will then apply in every
+record, which reference images.
```
Comment (web-commerce-og) [inline Documentation/ApiOverview/CropVariants/General/Index.rst:11]: in every record referencing images.

File: Documentation/ApiOverview/CropVariants/General/Index.rst [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+In this example we configure two crop variants, one with the id "mobile",
+one with the id "desktop". The array key defines the crop variant id, which will be used
+when rendering an image with the image view helper.
+
+The allowed crop areas are now also configured differently.
```
Comment (web-commerce-og) [inline Documentation/ApiOverview/CropVariants/General/Index.rst:17]: "now" as to compared with when and what?

File: Documentation/ApiOverview/CropVariants/General/Index.rst [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+=====================
+General Configuration
+=====================
+
+The following examples are basically meant to add one single
+cropping configuration to sys_file_reference,  which will then apply in every
+record, which reference images.
+
+In this example we configure two crop variants, one with the id "mobile",
+one with the id "desktop". The array key defines the crop variant id, which will be used
```
Comment (web-commerce-og) [inline Documentation/ApiOverview/CropVariants/General/Index.rst:14]: that would be the keys in allowedAspectRatios i.e. 4:3 and NaN?

File: Documentation/ApiOverview/CropVariants/General/Index.rst [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+when rendering an image with the image view helper.
```
Comment (web-commerce-og) [inline Documentation/ApiOverview/CropVariants/General/Index.rst:15]: how will it be used in the image view helper?

File: Documentation/ApiOverview/CropVariants/General/Index.rst [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+            ],
+        ],
+    ]
+
+Cover Area
+==========
+
+Very often images are used in a context, where there are overlaid with other DOM elements
+like a headline. To give editors a hint which area of the image is affected, when selecting a crop area,
+it is possible to define multiple so called cover areas. These areas are shown inside
```
Comment (web-commerce-og) [inline Documentation/ApiOverview/CropVariants/General/Index.rst:113]: so-called

**Review by web-commerce-og — APPROVED** [review]
Great work , thank you! I added some changes that would be nice if included but would also approve your work as-is

File: Documentation/ApiOverview/CropVariants/General/Index.rst (hunk 4 above)
Comment (h3nn3s) [reply Documentation/ApiOverview/CropVariants/General/Index.rst:15]: Please have a look at the end of the page:
`<f:image image="{data.image}" cropVariant="mobile" width="800" />`

File: Documentation/ApiOverview/CropVariants/General/Index.rst (hunk 2 above)
Comment (h3nn3s) [reply Documentation/ApiOverview/CropVariants/General/Index.rst]: "now" as compared to versions prior 8 LTS - removed it completely

File: Documentation/ApiOverview/CropVariants/General/Index.rst (hunk 3 above)
Comment (h3nn3s) [reply Documentation/ApiOverview/CropVariants/General/Index.rst:14]: yep, but maybe these keys are not relevant at all. 
They only define the preconfigured aspect ratios in backend UI, I think

**Comment by susannemoog** [comment]
thanks a lot for this :) I did some small wording / link updates - if there are more things unclear, we can do it in separate patches, this is already worthy of a release 👍

**Pull request merged**